core/execution/liquidation/snapshot.go

```diff
@@ -57,9 +57,13 @@ func (e *Engine) LoadState(ctx context.Context, pl *types.Payload) ([]types.Stat
 		if d.Config != nil {
 			e.cfg = d.Config.DeepClone()
 		} else {
-			// @NOTE this can be removed after protocol upgrade has completed
+			// this can probably be removed now
 			e.cfg = GetLegacyStrat()
 		}
+		// @NOTE this should have a protocol upgrade guard around it
+		if e.cfg.DisposalFraction.IsZero() {
```

Comment:
Had to read it again, it does say that... Not sure if that's really the way to go. If that's the case, then it definitely wouldn't make sense to also specify a default value of `0.1` for the slippage parameter (also in spec: https://github.com/vegaprotocol/specs/pull/2218/files#diff-3d81a39049a36bb18ecac5bd2f70835f6faabe4410a489eddc8f531411322bacR114)

Reply:
Probably worth clearing that up with protocol-design people then. Because also there being a "default" implies that the new field should be optional, whereas we have it here as a required parameter that must be set to something.

Reply:
https://github.com/vegaprotocol/specs/pull/2249

Default of 0.1 to be removed from spec, using SLA price range for proposals/existing markets now